src/Entity/ExampleProgram.php

```diff
@@ -0,0 +1,223 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\HttpFoundation\File\File;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\ExampleRepository")
+ * @ORM\EntityListeners({"App\Catrobat\Listeners\Entity\ExampleProgramImageListener"})
+ * @ORM\Table(name="example")
+ */
+class ExampleProgram
+{
+  /**
+   * @var File
+   */
+  public $file;
+  /**
+   * @var
+   */
+  public $removed_id;
+  /**
+   * @var
+   */
+  public $old_image_type;
+
+  /**
+   * @ORM\Id
+   * @ORM\Column(type="integer")
+   * @ORM\GeneratedValue(strategy="AUTO")
+   */
+  protected $id;
+
+  /**
+   * @ORM\Column(type="string")
+   */
+  protected $imagetype;
+
+  /**
+   * @ORM\ManyToOne(targetEntity="Program", fetch="EAGER")
+   **/
+  private $program;
+
+  /**
+   * @ORM\Column(type="string", nullable=true)
+   */
+  protected $url;
```

Comment:
for what is this url member used? I found no field to set it in the admin interface.